### ISSUE TITLE
Allow "Change Type" on multiple parts at once

### ISSUE THREAD

**Issue by FatHammer** [bambulab/BambuStudio#5603]
**Is your feature request related to a problem? Please describe.**
I sometimes want to change the types of multiple parts of an object at once (to negative, modifier, etc.), but having to do each one individually can get somewhat time-consuming.

**Describe the solution you'd like**
I would like to be able to select multiple parts at once, and change the type of them all at the same time.

**Describe alternatives you've considered**
Nothing really comes to mind.

**Additional context**
![image](https://github.com/user-attachments/assets/14af5b24-6b95-40d1-aa7b-cb5ef92de898)

![image](https://github.com/user-attachments/assets/09eb4a57-09f3-4ca7-af5c-2ea19344a7e6)

**Comment by badkernel** [comment]
I agree, this is something that should be added as soon as possible.

**Comment by vanowm** [comment]
Duplicate of [#1400](https://github.com/bambulab/BambuStudio/issues/1400)
